DrawerKit/DrawerKit/Internal API/DrawerState.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+enum DrawerState: Equatable {
+    case collapsed
+    case partiallyExpanded
+    case fullyExpanded
+    case transitioning(CGFloat) // current drawer Y position
+}
```

Comment:
`case transitioning(currentY: CGFloat)` is, at least in my opinion, clearer than adding a documenting comment.